app/geo/gui/src/ts/twxplore/gui/geo/components/mapPage.tsx/MapPage.tsx

```diff
@@ -0,0 +1,156 @@
+import * as React from "react";
```

Comment:
Directory names shouldn't have extensions.
Usual nomenclature for this type of component is a "layout". MapLayout.

Reply:
You could put MapLayout in components/map/, the component subdirectories are for functionally similar components.